pgcontents/checkpoints.py

```diff
@@ -65,7 +65,7 @@ def rename_checkpoint(self, checkpoint_id, old_path, new_path):
         """Rename a checkpoint from old_path to new_path."""
         with self.engine.begin() as db:
             return move_single_remote_checkpoint(
-                db, old_path, new_path, checkpoint_id,
+                db, self.user_id, old_path, new_path, checkpoint_id,
```

Comment:
Is this just broken in the current incarnation?  I thought we had tests for this case...